src/components/Navbar.jsx

```diff
@@ -1,31 +1,51 @@
 import React from 'react';
-import Link from 'next/link';
-import { Navbar, NavbarBrand, Nav } from 'reactstrap';
+import { Navbar, Nav } from 'reactstrap';
 import NavbarStyle from './NavbarStyle';
-// import Imgix from './Imgix';
+import { StoryContext } from './StoryProvider';
+import Imgix from './Imgix';
 // import { AccountContext } from './AccountProvider';
 
 export default () => (
   <Navbar expand fixed="top">
     <NavbarStyle />
-    <Link href="/" passHref>
-      <NavbarBrand>
-        {/* <AccountContext.Consumer>
-          {({ branding }) => {
+    <Nav className="position-absolute">
+      {/* <AccountContext.Consumer>
+        {({ branding }) => {
+          return (
+            <Imgix
+              className="d-flex bg-imgIx img-fluid"
+              src={branding.logo.url}
+              alt={branding.name}
+              title={branding.name}
+              w="250"
+              h="45"
+            />
+        ) }}
+      </AccountContext.Consumer> */}
+    </Nav>
+    <Nav className="mx-auto navbar-logo" navbar>
+      <StoryContext.Consumer>
+        {({ story }) => {
+          if (story.advertiser && story.advertiser.logo) {
             return (
               <Imgix
                 className="d-flex bg-imgIx img-fluid"
-                src={branding.logo.url}
-                alt={branding.name}
-                title={branding.name}
-                w="250"
-                h="45"
+                src={story.advertiser.logo.src}
+                alt={story.advertiser.name}
+                title={story.advertiser.name}
+                w="300"
```

Comment:
The w/h params will cause issues for non-landscape logos.

See this for a possible solution: https://github.com/cygnusb2b/fortnight-app/blob/ad44ee082b4f547df9ea62a72530db14053c5a2c/app/templates/manage/advertiser/list-item.hbs#L11